IBPSA/Examples/Tutorial/SimpleHouse/SimpleHouseTemplate.mo

```diff
@@ -0,0 +1,103 @@
+within IBPSA.Examples.Tutorial.SimpleHouse;
+model SimpleHouseTemplate
+  "Template file for simple house example"
+  extends Modelica.Icons.Example;
+  package MediumAir = IBPSA.Media.Air "Medium model for air";
+  package MediumWater = IBPSA.Media.Water "Medium model for water";
+  parameter Modelica.Units.SI.Area A_wall = 100 "Wall area";
+  parameter Modelica.Units.SI.Length d_wall = 0.25 "Wall thickness";
+  parameter Modelica.Units.SI.ThermalConductivity k_wall = 0.04 "Wall thermal conductivity";
+  parameter Modelica.Units.SI.Density rho_wall = 2000 "Wall density";
+  parameter Modelica.Units.SI.SpecificHeatCapacity cp_wall = 1000 "Wall specific heat capacity";
+  IBPSA.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(filNam=
+        ModelicaServices.ExternalReferences.loadResource(
+        "modelica://IBPSA/Resources/weatherdata/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"))
+    "Weather data reader"
+    annotation (Placement(transformation(extent={{-200,-20},{-180,0}})));
+  IBPSA.BoundaryConditions.WeatherData.Bus weaBus "Weather data bus"
+    annotation (Placement(transformation(extent={{-160,-20},{-140,0}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedTemperature Tout
```

Comment:
Should be `TOut`

Reply:
I updated all parameter and components names according to the naming convention. Please let me know if you still find conflicts with the naming convention.